### DISCUSSION TITLE
Saturn preset missing?

### DISCUSSION THREAD

**Discussion by Raton-Laveur** [darthcloud/BlueRetro#610] [Q&A] [answered]
Hello, 

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#saturn in "Mapping" section specifies a preset named "Saturn Merge analog & digital trigger", but it's absent from https://blueretro.io/presets.html : 

![image](https://user-images.githubusercontent.com/17532038/219457076-37ec4254-1085-44cb-87d7-4b47d3cd0ff8.png)

Am I looking in the wrong place?

Slightly unrelated, but I'd like to contribute to the project wiki and it looks like it's only editable by project members. Can I apply for wiki edit privilege?

Thanks!

**Comment by darthcloud** [comment, marked as answer]
The trigger merge is now handled by the FW itself for the 3D controller mode.

So the preset was removed since not required any more.

I updated the wiki to remove outdated info.

GitHub have no granularity for write privilege. So no for the official one.
But I just setup a mirror at https://github.com/darthcloud/BlueRetroWiki where you can send PR for wiki edit more easily.
